README.md

```diff
@@ -59,7 +59,7 @@ test/data/mustermann_2021_svgdigitizer_1_p2.svg
 test/data/mustermann_2021_svgdigitizer_1_p3.png
 test/data/mustermann_2021_svgdigitizer_1_p3.svg
 
-$ svgdigitizer cv doc/files/mustermann_2021_svgdigitizer_1/mustermann_2021_svgdigitizer_1_f2a_blue.svg
+$ svgdigitizer cv doc/files/mustermann_2021_svgdigitizer_1/mustermann_2021_svgdigitizer_1_f2a_blue.svg # byexample: +skip

```

Comment:
A number of warnings are printed which do not appear in the code block. I don't think testing is very useful here since we test all kinds of files in the test folder.

Reply:
What kind of warnings? Typically, these things break eventually if they're not tested.

Reply:
These are deprecation warnings which originate from astropy. I need to find out where they stem from.